api/v1/nificluster_types.go

```diff
@@ -582,7 +582,7 @@ type LdapConfiguration struct {
 	// Password for the Keystore that is used when connecting to LDAP using LDAPS or START_TLS.
 	TLSKeystorePassword string `json:"tlsKeystorePassword,omitempty"`
 	// Type of the Keystore that is used when connecting to LDAP using LDAPS or START_TLS (i.e. JKS or PKCS12).
-	TLSKeystoretype string `json:"tlsKeystoreType,omitempty"`
+	TLSKeystoreType string `json:"tlsKeystoreType,omitempty"`
```

Comment:
There was a successful build with this typo? Is it used??

Reply:
Ah, i see the slack thread. This is probably something we should unit test so we don't discover issues like this after release.